yarn.lock

```diff
@@ -2223,6 +2231,18 @@
     "@types/react-reconciler" "^0.26.0"
     react-reconciler ">=0.26.0 <0.27.0"
 
+"@remote-ui/react@^4.5.2":
```

Comment:
Could we dedupe `@remote-ui/{core,react}`? I don't think the duplicate versions are necessary in this case.

Reply:
Good catch. We should probably add a check to CI before we have too many duplicates.